docs/developers_guide.md

```diff
@@ -318,3 +318,7 @@ Wed May 24 02:04:40 2023
 +---------------------------------------------------------------------------------------+
 
 ```
+
+## Debug Mode
+
+When `DebugMode` in `configs/setup/system.json` is set to true, we add the `--v=5` parameter to set the log level verbosity of kubelet to 5. In addition, we also execute the `IncreaseLogSizePerContainer` function to increase the LogSize of the Container. The default value of `DebugMode` is false.
```

Comment:
I'd say, make it more high level, like "In order to simplify the debugging procedure of whole created cluster, `DebugMode` parameter in `configs/setup/system.json` can be set to true. This debug mode increases the amount of log messages from the kubelet and increases the maximum size of stored logs from containers. By default, this mode is disabled for performance reasons."